### DISCUSSION TITLE
[OSPP-Week2] Carry on building the framework and implement CRR algorithm

### DISCUSSION THREAD

**Discussion by pilgrimygy** [JuliaReinforcementLearning/ReinforcementLearning.jl#363] [Archive]
@findmyway 
[Last week plan](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/discussions/344)

Last week, I did the following work:
1. Build a simple offline framework and CartPole datasets (including different random behavior policies).
2. Test the offline DQN algorithm.
3. Write a simple CQL loss component. After testing, it can greatly improve the performance of offline DQN.

Week 2:
This week, I will try to complete the framework and implement the Critic Regularized Regression (CRR) algorithm. I wanna implement discrete and continuous versions.

**Comment by pilgrimygy** [comment]
If implementing continuous CRR, maybe I need to write an offline AC algorithm as the benchmark and make a Pendulum Dataset.

**Reply by pilgrimygy** [reply]
I think that maybe we need to directly use the written online algorithm as an offline benchmark instead of rewriting it. @findmyway

**Reply by findmyway** [reply]
Agree

**Reply by pilgrimygy** [reply]
I want to know where you think we should put the dataset.

**Reply by findmyway** [reply]
Just create a new package under `src/` folder named `ReinforcementLearningDatasets`
